src/hooks/useTwitchApi.ts

```diff
@@ -27,3 +27,16 @@ export const useTwitchApi = <T>(path: string | null) => {
     refreshInterval: 10000,
   });
 };
+
+export const useLegacyTwitchApi = <T>(path: string | null) => {
```

Comment:
I preferred making a separate function as this has its own quirks we don't want to keep into the "healthy" code using the new API. Hopefully we can just remove this sooner than later.